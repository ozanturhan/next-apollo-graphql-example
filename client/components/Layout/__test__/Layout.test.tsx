import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import { Layout } from '../Layout';

it('renders correctly', () => {
  const tree = render(<Layout />).container;
  expect(tree).toMatchSnapshot();
});
