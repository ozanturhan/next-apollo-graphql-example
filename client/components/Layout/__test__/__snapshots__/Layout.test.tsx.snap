// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

@media only screen and (max-width:1200px) {
  .c0 {
    height: 100%;
    padding: 10px;
  }
}

<div>
  <div
    class="c0"
  />
</div>
`;
